h/admin/views/features.py

```diff
@@ -33,5 +34,30 @@ def features_save(request):
         location=request.route_url('admin_features'))
 
 
+@view_config(route_name='admin_cohorts',
+             request_method='GET',
+             renderer='h:templates/admin/cohorts.html.jinja2',
+             permission='admin_features')
+@paginator.paginate
+def cohorts_index(context, request):
+    query = request.db.query(models.FeatureCohort)
+    return query.order_by(models.FeatureCohort.name)
+
+
+@view_config(route_name='admin_cohorts',
+             request_method='POST',
+             request_param='add',
+             renderer='h:templates/admin/cohorts.html.jinja2',
+             permission='admin_features')
+def cohorts_add(request):
+    """Create a new feature cohort."""
+    cohort_name = request.params['add']
```

Comment:
Might be good to not allow creating cohorts with an empty name. I'll merge this anyway as it's not the end of the world :)